src/get_ensemble_output.R

```diff
@@ -3,41 +3,69 @@ library(hubVis)
 library(hubEnsembles)
 library(hubData)
 library(dplyr)
+library(purrr)
+library(here)
 
-
-hub_path <- "/Users/dk29776/Dropbox/UTAustin/flu-metrocast"
+hub_path <- here("..", "flu-metrocast")
```

Comment:
I'm not an expert in how `here()` works, but I was thinking that just using 
```suggestion
hub_path <- here()
```
would be the better way to fix this.